.buildkite/rllib_contrib.rayci.yml

```diff
@@ -2,10 +2,28 @@ group: rllib contrib tests
 depends_on:
   - oss-ci-base_build
 steps:
-  - label: ":exploding_death_star: rllib contrib: a2c tests"
+  - label: ":exploding_death_star: rllib contrib: {{matrix}} tests"
```

Comment:
can we change the `exploding_death_star` to something else that is not animating?